### ISSUE TITLE
License is incorrectly identified as LGPL

### ISSUE THREAD

**Issue by fishface60** [JasonMahdjoub/UPnPIGD#3]
The COPYING file is LGPL.

The README.md says LGPL or optionally CDDL.

The last version of Cling was LGPL or CDDL.

_However_ I see from the commit history you have been merging commits from jupnp into this project. Unless these are all your own work or you have been getting permission from the authors to license those changes as LGPL the only legal way to use this code is as CDDL, so the whole project is effectively CDDL now.

**Comment by JasonMahdjoub** [comment]
You are right. I am new to maintaining this code. I started with Cling and got confused when I used some portions of the JUPNP code. The only way to make the project compatible with LGPL is to rebase the code from Cling and ignore the modifications introduced by JUPNP. Therefore, I will need to overwrite the main branch. Do you agree with this approach?

**Comment by JasonMahdjoub** [comment]
I precise that very little portions of code was used. All other portions of modifications was my work.

**Comment by JasonMahdjoub** [comment]
Finally, I have rebased the "main" branch to an older version that no longer uses JUPNP code. As a result, this repository can now be licensed under LGPL-v2.1. Apologies for overriding the "main" branch.
